css/css-flexbox_navigation/css/styles.css

```diff
@@ -29,3 +29,19 @@ ul {
   padding: 5px 14px;
   border-radius: 5px;
 }
+
+.navigation__menu {
+  display: flex;
+  justify-items: center;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.navigation__link .first {
+  align-items: flex-start;
+}
+
+.navigation__link .last {
+  align-items: flex-end;
+}
```

Comment:
`.first` and `.last` sound like they are modifier classes. In this exercise it doesn't matter so much but in the future you might want to avoid these arbitrary class names that may cause other elements to have their stiles overriden. Hence, you can be even more specific with your selector by using the full extent of the BEM convention and selecting with something like `.navigation__link--first` and `.navigation__link--last`

Reply:
ps: you've left a space between `.navigation__link`and `.first` (and `.last`) which means you are targetting the children of `navigation__link` which have the `.first` (or `.last`) class, not the element itself, which is what I believe you wanted.

Reply:
All that being said, I think both of those `.navigation__link.first` and `.navigation__link.first`are unnecessary anyways. 😅